content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -31,6 +25,30 @@ Supported Agent versions
 Data privacy
 : Enabling SQL comment propagation results in potentially confidential data (service names) being stored in the databases which can then be accessed by other third-parties that have been granted access to the database.
 
+
+**Supported tracers**
+
+| Language | Library                        | Postgres    | MySQL       |
```

Comment:
I think this looks great but one thing that came up as we were doing the support matrix was the `library` terminology. I think maybe `library` is the most sensible term but some of the items in that column are actually drivers. I thought about using the generic term `technology` at some point but it's probably imperfect as well. Did you check how the rest of documentations refers to those _things_? I think as long as we align with that, this should be good.

Reply:
i did not - great callout, i'll look around other docs.

Reply:
oof we basically use different words for different languages. `library` and `framework` are the two most common. It seems like in the land of tracers, we don't reference `drivers` at all. I can update the column name to `Library or Framework`?

Reply:
Yeah, that sounds good to me. Thanks for checking!